### PR TITLE
v0.13.0 release note

### PR DESCRIPTION
**Pull request by abuccts**
WIP

### CODE REVIEW

File: RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Release v0.13.0
+
+
+## New Features
+
+* PAI protocol:
+  - Introduce PAI protocol and job submission v2 [#2260](https://github.com/microsoft/pai/pull/2260)
+  - Add new job submission v2 plugin [#2461](https://github.com/microsoft/pai/pull/2461)
+  - Add new marketplace plugin, support GitHub and Azure DevOps repository [#2703](https://github.com/microsoft/pai/pull/2703), [#2724](https://github.com/microsoft/pai/pull/2724)
```
Comment (scarlett2018) [inline RELEASE_NOTE.md]: Let me discuss with leads on whether we should release this marketplace plugin in May release first.

Comment (abuccts) [reply RELEASE_NOTE.md]: removed

File: RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Release v0.13.0
+
+
+## New Features
+
+* PAI protocol:
```
Comment (scarlett2018) [inline RELEASE_NOTE.md]: PAI --> OpenPAI. this comment apply to the rest places in the entire .md file.

File: RELEASE_NOTE.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  - Introduce PAI protocol and job submission v2 ([#2260](https://github.com/microsoft/pai/pull/2260))
```
Comment (scarlett2018) [inline RELEASE_NOTE.md]: Do you want to differ this "V2" with the previous "V2"?

Comment (scarlett2018) [reply RELEASE_NOTE.md]: can you add the Protocol V2 introduction document (link) here?

File: RELEASE_NOTE.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  - Add new job submission v2 plugin ([#2461](https://github.com/microsoft/pai/pull/2461))
+
+* Web portal:
+  - Add new home page ([#2544](https://github.com/microsoft/pai/pull/2544), [#2614](https://github.com/microsoft/pai/pull/2614))
```
Comment (scarlett2018) [inline RELEASE_NOTE.md]: Find Yang or Qinzheng to add a screen shot and simple introduction here?

File: RELEASE_NOTE.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  - Add new user management page ([#2726](https://github.com/microsoft/pai/pull/2726), [#2796](https://github.com/microsoft/pai/pull/2796))
```
Comment (scarlett2018) [inline RELEASE_NOTE.md]: User Management UX refactoring with new layout and themes.

File: RELEASE_NOTE.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+
+## Improvements
+
+* PAI protocol:
+  - Update example jobs in marketplace v2 for PAI protocol ([#2827](https://github.com/microsoft/pai/pull/2827))
```
Comment (scarlett2018) [inline RELEASE_NOTE.md]: remove this line?

File: RELEASE_NOTE.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+* Web portal:
+  - Refine styles in job pages ([#2829](https://github.com/microsoft/pai/pull/2829))
+  - Refine alert message in job pages ([#2698](https://github.com/microsoft/pai/pull/2698))
+  - Reduce build bundle size ([#2715](https://github.com/microsoft/pai/pull/2715))
```
Comment (scarlett2018) [inline RELEASE_NOTE.md]: what's this feature for?

File: RELEASE_NOTE.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+* Rest server:
+  - Check duplicate job in submission v2 ([#2837](https://github.com/microsoft/pai/pull/2837))
+
+* Hadoop:
+  - Increase YARN kill container timeout ([#2778](https://github.com/microsoft/pai/pull/2778))
+  - Remove cross origin in resource manager ([#2758](https://github.com/microsoft/pai/pull/2758))
+  - Fix Haddoop AI matching nvidia-smi regex ([#2681](https://github.com/microsoft/pai/pull/2681))
+
+
```
Comment (scarlett2018) [inline RELEASE_NOTE.md]: May you check whether all the following planned items are all listed?
https://github.com/microsoft/pai/milestone/3

File: RELEASE_NOTE.md (hunk 2 above)
Comment (abuccts) [reply RELEASE_NOTE.md]: updated

File: RELEASE_NOTE.md (hunk 3 above)
Comment (abuccts) [reply RELEASE_NOTE.md]: no, previous "v2" has been removed
link added

File: RELEASE_NOTE.md (hunk 5 above)
Comment (abuccts) [reply RELEASE_NOTE.md]: updated

File: RELEASE_NOTE.md (hunk 6 above)
Comment (abuccts) [reply RELEASE_NOTE.md]: this is for protocol examples, not marketplace plugin

File: RELEASE_NOTE.md (hunk 7 above)
Comment (abuccts) [reply RELEASE_NOTE.md]: this is an improvement, not a feature. it makes building and visiting webportal faster

File: RELEASE_NOTE.md (hunk 8 above)
Comment (abuccts) [reply RELEASE_NOTE.md]: many items in https://github.com/microsoft/pai/milestone/3 are not in this release, e.g. AAD account, user group, user storage, etc.
aren't the planned items in this release listed in #2386?

File: RELEASE_NOTE.md (hunk 7 above)
Comment (scarlett2018) [reply RELEASE_NOTE.md]: Okey, let's change it to: Reduce the build bundle size to improve webportal performance.

Comment (abuccts) [reply RELEASE_NOTE.md]: updated

**Pull request merged**